frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/StatusCard.tsx

```diff
@@ -66,7 +64,11 @@ const getHostHardwareHealthState = (obj): HostHealthState => {
 const filterAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'labels.hwalert'));
 
-const HealthCard: React.FC<HealthCardProps> = ({ watchAlerts, stopWatchAlerts, alertsResults }) => {
+const HealthCard: React.FC<HealthCardProps> = ({
+  watchAlerts,
```

Comment:
Where is `watchAlerts` defined?  It seems to have been removed from `with-dashboard-resources.tsx`?